ucm2/USB-Audio/Digidesign/Digidesign-Mbox-3-HiFi.conf

```diff
@@ -39,55 +39,56 @@ Macro [
 	}
 ]
 
-SectionDevice."Line1" {
-	Comment "Main Output L/R"
-
+SectionDevice."line2SPDIF" {
+	Comment "SPDIF Out"
```

Comment:
I'm ok with the reoder in this file, but line2SPDIF does not follow naming rules. Use just Line2 and describe the device in the comment.